### PR TITLE
Wishlist & Cart management

### PR DESCRIPTION
**Pull request by shubhmehra**
Wishlist manager updates

### CODE REVIEW

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]
Wishlist link from Homepage not working. It works from cart though.
Else It seems fine.

**Comment by dusariudaykumar** [comment]
Unable to open the wishlist page

File: style.css [hunk 1]
```diff
@@ -1,10 +1,62 @@
...
+  max-width: 24rem;
+}
+
+.e-comm-card {
+  gap: 64px;
+}
+
+.card-text-gap {
+  margin-bottom: 20px;
+}
```
Comment (dusariudaykumar) [inline style.css]: Just one suggestion you can use rem's here as well.

**Review by dusariudaykumar — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (shubhmehra) [reply style.css]: resolved

**Pull request merged**